examples/incredible-squaring/config/config.toml

```diff
@@ -26,6 +24,14 @@ quorum_threshold_percentage = 100
 
 quorum_number = [0]
 
+[bls_signer]
+
+keystore_path = "keys/test.bls.key.json"
+
+keystore_password = ""
+
+private_key = "0x2518600ef40ef39cb4ab8b828ce303b3e02ac01ec6ba6bd0d0cf0663e1252ff0"
```

Comment:
Let's keep one of the examples using only the private key option and another one using only the keystore options.